### PR TITLE
Feature/additional service parameters

### PR DESCRIPTION
**Pull request by alxdsz**
Hello,

Change allows passing additional parameters and secrets to the deployed service.
I think I don't have permissions to attach reviewers here..

Cheers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,23 @@ Finally if you like to add DataDog Monitoring for the included service append th
 
 See the [configuration](#Configuration) section for a detailed overview of parameters.
 
+### Define additional parameters for deployed services
```
Comment (mithie) [inline README.md:101]: I would reformulate it a bit:

In such cases an additional ConfigMap or Secret will be created to pass the necessary values to the container as environmental variables. To do that you can include a supplementary values.yaml file in your own project and format it like in the code snippet below:

File: README.md [hunk 2]
```diff
@@ -98,6 +98,23 @@ Finally if you like to add DataDog Monitoring for the included service append th
...
+  secretsMapApplied: true # flag to tell secrets are defined
+  config:
+    <PARAM_NAME>: <PARAM_VALUE>
+    <PARAM_NAME2>: <PARAM_VALUE2>
+    ...
+  secret:
+    <SECRET_NAME>: <SECRET_VALUE>
+    ...
+```
+
```
Comment (mithie) [inline README.md:117]: Here I would add a short description on how to call the chart when an additional values.yaml file has been included in a project, i.e.

You can then call the chart by passing the additional values.yaml as command line argument:

```
helm install ...
```

File: README.md (hunk 1 above)
Comment (alxdsz) [reply README.md:101]: done

File: README.md (hunk 2 above)
Comment (alxdsz) [reply README.md:117]: done

File: templates/secret.yaml [hunk 3]
```diff
@@ -6,8 +6,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   {{- if .Values.project.includeAwsCredentials }}
-  aws.accesskey: {{ .Values.secret.aws_accesskey }}
-  aws.secretkey: {{ .Values.secret.aws_secretkey }}
+  aws.accesskey: {{ .Values.secret.aws_accesskey | b64enc }}
```
Comment (mithie) [inline templates/secret.yaml:9]: please remove the base64 encoding. variables need to be already base64 encoded when passed

File: templates/secret.yaml [hunk 4]
```diff
@@ -19,8 +19,8 @@ metadata:
   name: {{ .Values.tls.cert.int.secret.name }}
   namespace: {{ .Release.Namespace }}
 data:
-  tls.crt: {{ .Values.tls.cert.int.secret.crt }}
-  tls.key: {{ .Values.tls.cert.int.secret.key }}
+  tls.crt: {{ .Values.tls.cert.int.secret.crt | b64enc}}
```
Comment (mithie) [inline templates/secret.yaml:22]: please remove the base64 encoding. variables need to be already base64 encoded when passed

**Pull request merged**